terrautils/gdal.py

```diff
@@ -68,3 +69,17 @@ def centroid_from_geojson(geojson):
 def wkt_to_geojson(wkt):
     geom = ogr.CreateGeometryFromWkt(wkt)
     return geom.ExportToJson()
+
+
+def find_plots_intersect_boundingbox(bounding_box, all_plots):
+    bbox_poly = ogr.CreateGeometryFromJson(str(bounding_box))
+    intersecting_plots = []
```

Comment:
please make intersecting_plots a dictionary {} then on line 84, set intersecting_plots[plotname] = all_plots[plotname] so we don't have to maintain a list of plot names in addition to the geometry dict, and we can just use this function output for that.

Reply:
done